### PR TITLE
Feature/dx 751

### PR DESCRIPTION
**Pull request by shikhanansi**
I've made the following changes:
1) Removed "draft" from Tiny Drive pages
2) Grammar edits
3) Renamed files from Tiny Comments and Tiny Drive to Comments and Drive. 
- Please note I've only renamed the files that affected the website navigation menu. 
- I've removed the later instances of `Tiny` from Tiny Drive and Tiny Comments after the first introduction.
4) Added a para explaining the user level in JWT doc
5) Some more edits
Please review.
Regards,
Shikha

### CODE REVIEW

File: configure/jwt-authentication.md [hunk 1]
```diff
@@ -1,23 +1,29 @@
...
+layout: default
 title: JWT Authentication
 description_short: JWT Authentication
 description: JWT is a common authorization solution for web services.
 ---
 
-Some cloud services for TinyMCE require you to setup JWT authentication. This allows us to verify that you and your end user are allowed to access a particular feature. JWT is a common authorization solution for web services and is documented in more detail at the https://jwt.io/ website. This guide aims to show how to setup JWT authentication for the cloud services provided for TinyMCE.
+### Audience
+
+This section is intended to be used by developers with prior knowledge of JSON Web Tokens (or JWTs) in detail, including how they can be used for User Authentication and Session Management in a Web Application. There will be some coding involved on both the client-side and the server-side to configure JWT as per the instructions in this section.
```
Comment (ephox-dayne) [inline configure/jwt-authentication.md]: s/(or JWTs)/(JWT)/

File: plugins/comments.md [hunk 2]
```diff
@@ -1,34 +1,34 @@
...
 
-Here is a demo to showcase the Tiny Comments functionality using storage configured to persist in real-time:
+Here is a demo to showcase the Comments functionality using storage configured to persist in real-time:
 
 {% include codepen.html id="pOzxJw" %}
 
 ### Storage - persist on content-save
 
-Here is a demo to showcase the Tiny Comments functionality using storage configured to persist on content-save.
+Here is a demo to showcase the Comments functionality using storage configured to persist on content-save.
```
Comment (tinydylan) [inline plugins/comments.md]: "Here is a demo showcasing"? 
"Here is a demo which showcases"?
"The following demo showcases"?

File: plugins/drive.md [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
+title: Drive
 title_nav: Drive
 description: Cloud-based file and image management for TinyMCE.
 keywords: tinydrive storage media tiny drive
 ---
 
 The Tiny Drive plugin adds the functionality to upload and manage files and images to the cloud. This plugin is only available in [Tiny Cloud](https://www.tiny.cloud/download/) and requires you to register for an API key.
 
-To enable this functionality, add `tinydrive` to the list of plugins in the `tinymce.init` call. You also need to authenticate the user using a [JWT token]({{site.baseurl}}/configure/jwt-authentication).
+To enable this functionality, add `tinydrive` to the list of plugins in the `tinymce.init` call. You also need to authenticate the user using a [JSON Web token]({{site.baseurl}}/configure/jwt-authentication).
```
Comment (tinydylan) [inline plugins/drive.md]: Capitalization: "JSON Web Token" ?

File: configure/jwt-authentication.md (hunk 1 above)
Comment (androb) [reply configure/jwt-authentication.md]: User Authentication, Session Management, and Web Application should not be capitalized in this case.

File: configure/jwt-authentication.md [hunk 4]
```diff
@@ -1,23 +1,29 @@
...
+
+## Introduction
+
+Some cloud services for TinyMCE require you to setup JSON web tokens authentication. This allows us to verify that you and your end user are allowed to access a particular feature. JWT is a common authorization solution for web services and is documented in more detail at the https://jwt.io/ website. This guide aims to show how to setup JWT authentication for the cloud services provided for TinyMCE.
```
[the remark this answers is not in the document]
Comment (shikhanansi) [reply configure/jwt-authentication.md]: Done.

File: configure/jwt-authentication.md [hunk 5]
```diff
@@ -58,7 +64,7 @@ try {
 
 ## Node Token Provider Example
 
-This example shows you how to set up a Node.js express handler that produces the JWT tokens. It requires you to install the Express web framework and the JSON web token Node modules. Each service requires different claims to be provided. The following example shows the sub and name claims needed for Tiny Drive.
+This example shows you how to set up a Node.js express handler that produces the JWT. It requires you to install the Express web framework and the JSON web token Node modules. Each service requires different claims to be provided. The following example shows the sub and name claims needed for Tiny Drive.
```
[the remark this answers is not in the document]
Comment (shikhanansi) [reply configure/jwt-authentication.md]: Done.

**Comment by shikhanansi** [comment]
Applied all review changes. Added @tjdett's updates from https://github.com/tinymce/tinymce-docs/pull/755.

**Pull request merged**